src/game/mod.rs

```diff
@@ -9,10 +9,10 @@
 use music;
 use opengl_graphics::GlGraphics;
 use opengl_graphics::glyph_cache::GlyphCache;
-use piston_window::{Button, clear, Context, Input, Key, PistonWindow, Size, text, Transformed,
```

Comment:
Hmmm, weird, rustfmt appears to care about ordering of imports now. Looks like capitalization matters.